certified-connectors/ProcessStreet/apiDefinition.swagger.json

```diff
@@ -2,16 +2,16 @@
   "swagger": "2.0",
   "info": {
     "title": "Process Street",
-    "description": "You can perform various actions such as starting, finding, and updating a workflow run in Process Street. Or trigger actions when a workflow starts or task status changes.",
-    "version": "1.0",
+    "description": "Process Street is a modern process management platform for teams. You can use this connector to run workflows, find and update workflow runs, and trigger actions when a workflow is run or changes.",
+    "version": "1.1.5",
```

Comment:
@mwysokinski you can keep the version as 1.0 if the connector hasn't been published yet or is in certification. Please keep it in `x.x` pattern.

Reply:
fixed